tarantool_test.go

```diff
@@ -1957,6 +1957,12 @@ func TestSchema(t *testing.T) {
 	if field2.Type != "string" {
 		t.Errorf("field 2 has incorrect Type")
 	}
+	if field_nullable, ok = space.Fields["nullable"]; !ok {
+		t.Errorf("field name = nullable was not found")
+	}
+	if !field_nullable.IsNullable {
+		t.Errorf("field nullable has incorrect IsNullable")
+	}
```

Comment:
I would like two changes:
1. Let's check other fields to make the test case obvious. Something like this or a [table + subtests](https://go.dev/blog/subtests):
```suggestion
        for i := 0; i <= 5; i++ {
        	name := space.Fields[fmt.Sprintf("name%d", i)]
		if field_nullable, ok = space.Fields[name]; !ok {
			t.Errorf("field name = %s was not found", name)
		}
		if field_nullable.IsNullable {
			t.Errorf("field %s has incorrect IsNullable", name)
		}
	}
	if field_nullable, ok = space.Fields["nullable"]; !ok {
		t.Errorf("field name = nullable was not found")
	}
	if !field_nullable.IsNullable {
		t.Errorf("field nullable has incorrect IsNullable")
	}
```

2. Let's move the code into a separate test - `TestSchema_IsNullable`. It will help us not to overcomplicate the current test.

Reply:
1. Used your suggestion (with small changes). Using subtests here looks too complicated for me (naming of our fields is simple and we can easily format it on the fly, like in your code).
2. Fixed